### ISSUE TITLE
Icon "happy" maps to non-existent "md-happy-outline" on android

### ISSUE THREAD

**Issue by lucaslcode** [GeekyAnts/NativeBase#1331]
## react-native, react and native-base version
```
    "native-base": "^2.3.1",
    "react": "16.0.0-alpha.12",
    "react-native": "https://github.com/expo/react-native/archive/sdk-21.0.2.tar.gz",
```
## Expected behaviour
NBIcons.json line 1402:
```
"happy": {
    "android": {
      "default": "md-happy-outline",
```
This causes a warning and wrong icon when using <Icon name="happy"/> on android.
## Actual behaviour
NBIcons.json line 1402:
```
"happy": {
    "android": {
      "default": "md-happy",
```
## Steps to reproduce (code snippet or screenshot)
Use <Icon name="happy"/> on android
## Is the bug present in both ios and android or in any one of them?
Android

**Comment by akhil-geekyants** [comment]
Made a PR #1358 for this issue

**Comment by SupriyaKalghatgi** [comment]
Fixed with NativeBase [2.3.5](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.5)
